### PR TITLE
docs:CHARTS-4672 add interactive filtering docs link

### PR DESCRIPTION
**Pull request by kristinamongo**
- Add a few links to the Interactive Filtering MongoDB docs in the IF guides
- Some automatic text prettifying (mainly removing extra spacing)

### CODE REVIEW

File: examples/readme.md [hunk 1]
```diff
@@ -20,11 +20,12 @@ Our **authenticated** examples are great references if you need help getting sta
 - [Google](https://github.com/mongodb-js/charts-embed-sdk/blob/master/examples/authenticated-google)
 
 If you'd like to build **interactive** charts, we have two examples showing this capability:
-  - [Getting started with click events (BETA)](https://github.com/mongodb-js/charts-embed-sdk/tree/master/examples/click-events-basic)
-  - [Interactive filtering using click events (BETA)](https://github.com/mongodb-js/charts-embed-sdk/tree/master/examples/click-events-filtering)
+
+- [Getting started with click events (BETA)](https://github.com/mongodb-js/charts-embed-sdk/tree/master/examples/click-events-basic)
+- [Interactive filtering using click events (BETA)](https://github.com/mongodb-js/charts-embed-sdk/tree/master/examples/click-events-filtering)
```
Comment (jarjee) [inline examples/readme.md:25]: I don't think this is `BETA` anymore, right? @kirangp ?

**Review by jarjee — APPROVED** [review]
This looks good, although given the huge amount of copy / paste I wonder if we should explore templating these markdown files.

File: examples/readme.md (hunk 1 above)
Comment (kristinamongo) [reply examples/readme.md:25]: Yes, good pick, will remove the beta label. Thankfully the npm packge in the Interactive Filtering examples is the latest one (not the beta) so it's just the label. :)

**Comment by kristinamongo** [comment]
Templating parts of the guide steps is a good idea @jarjee. Will log a doc ticket 👍

**Pull request merged**